editor/tests/org.fusesource.ide.camel.editor.tests/src/test/java/org/fusesource/ide/camel/editor/component/wizard/ComponentGroupedByTagsTreeContenProviderTest.java

```diff
@@ -1,39 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2016 Red Hat, Inc. 
- * Distributed under license by Red Hat, Inc. All rights reserved. 
- * This program is made available under the terms of the 
- * Eclipse Public License v1.0 which accompanies this distribution, 
- * and is available at http://www.eclipse.org/legal/epl-v10.html 
- * 
- * Contributors: 
- * Red Hat, Inc. - initial API and implementation 
- ******************************************************************************/
-package org.fusesource.ide.camel.editor.component.wizard;
-
-import org.fusesource.ide.camel.tests.util.CamelModelRegistrationRule;
-import org.junit.Rule;
-import org.junit.Test;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class ComponentGroupedByTagsTreeContenProviderTest {
```

Comment:
because it was fully incompatible because of usage of CamelModelRegistrationRule. I will try to provide a replacement.